lib/command/src/Obelisk/Command/Run.hs

```diff
@@ -45,11 +48,13 @@ import System.Environment (getExecutablePath)
 import System.FilePath
 import qualified System.Info
 import System.IO.Temp (withSystemTempDirectory)
+import System.Process (waitForProcess)
```

Comment:
We should never be importing `System.Process` directly since we wrap the interface. See previous comment for the fix.